src/nix/sigs.cc

```diff
@@ -72,15 +72,17 @@ struct CmdCopySigs : StorePathsCommand
                         if (!info->sigs.count(sig))
                             newSigs.insert(sig);
                 } catch (InvalidPath &) {
+                    printError("path %s was invalid in substituter %s", storePath.to_string(), store2->getUri());
```

Comment:
```suggestion
                    debug("path %s was invalid in substituter %s", storePath.to_string(), store2->getUri());
```

I don't think we want to throw that at people's faces since having a missing path in a substituter is a completely normal situation